### PR TITLE
Update templates for migration.

### PR DESCRIPTION
**Pull request by adiroiban**
This is the content of the new templates for PR and ISSUE during the migration.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Ongoing migration
+=================
+
+Please don't create an issue yet.
```
Comment (adiroiban) [inline .github/ISSUE_TEMPLATE]:
```suggestion
Please DO NOT create an issue yet.
```

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -1,40 +1,9 @@
-<!--
-## Remove this section
+Ongoing migration
+=================
 
-Have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
+Please don't create a pull request yet.
```
Comment (adiroiban) [inline .github/PULL_REQUEST_TEMPLATE]:
```suggestion
Please DO NOT create a pull request yet.
```

**Review by adiroiban — COMMENTED** [review]
suggestions from @graingert

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+We are in the process of migrating the repository.
+
+The migration will be done in 1 day and after that you can create a PR.
```
Comment (adiroiban) [inline .github/ISSUE_TEMPLATE]:
```suggestion
The migration should be done by 28 June 2022, 13:00 UTC.
After that, you can create an issue.
```

File: .github/PULL_REQUEST_TEMPLATE [hunk 4]
```diff
@@ -1,40 +1,9 @@
...
+We are in the process of migrating the repository.
 
-Note that the Trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small;
-if you don't do them, it's likely that nobody will even notice your PR needs a review.
--->
+The migration will be done in 1 day and after that you can create a PR.
```
Comment (adiroiban) [inline .github/PULL_REQUEST_TEMPLATE]:
```suggestion
The migration should be done by 28 June 2022, 13:00 UTC.
After that, you can create a PR.
```

**Review by adiroiban — COMMENTED** [review]
more suggestions

**Review by danuker — APPROVED** [review]
Looks fine to me!

Progress is tracked here: https://github.com/chevah/trac-to-github/issues/16

**Pull request merged**